### ISSUE TITLE
Create Universe is failing with Time Out Exception

### ISSUE THREAD

**Issue by sraghavan-eqix** [YugaByte/yugabyte-db#818]
Hi Team,
Tried to create Universe via Yugaware and this fails with the below error message;
Failed to execute task java.util.concurrent.FutureTask@27b350ee, hit error java.lang.RuntimeException: java.lang.RuntimeException: Timed out waiting for Master Leader.. 

**Yugabyte Version: 1.1.9.0** 

STeps to replicate:
Created Yugaware Pod using Helm chart provided
SSH'ed into POD to disable Load Balancer Service as this is not supported. 
This is the file we disabled under opt/yugabtye/yugware/conf -> k8s-expose-all.yml

serviceEndpoints:
```
#  - name: "yb-master-service"
#    type: "LoadBalancer"
#    app: "yb-master"
#    ports:
#      ui: "7000"

#  - name: "yb-tserver-service"
#    type: "LoadBalancer"
#    app: "yb-tserver"
#    ports:
#      ycql-port: "9042"
#      yedis-port: "6379"
```

When Create Universe is invoked we see the namespace being created and the pods are running
```
kubectl get pods -n yb-dev-test
NAME           READY     STATUS    RESTARTS   AGE
yb-master-0    1/1       Running   0          1h
yb-tserver-0   1/1       Running   2          1h
yb-tserver-1   1/1       Running   2          1h
yb-tserver-2   1/1       Running   2          1h

kubectl get services -o wide -n yb-dev-test
NAME          TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)                               AGE       SELECTOR
yb-masters    ClusterIP   None         <none>        7100/TCP,7000/TCP                     1h        app=yb-master
yb-tservers   ClusterIP   None         <none>        7100/TCP,9000/TCP,6379/TCP,9042/TCP   1h        app=yb-tserver
```

**Comment by sraghavan-eqix** [comment]
Full error logs:

2019-01-29 23:46:13.372 ERROR AsyncYBClient.java:1921 [application-akka.actor.default-dispatcher-151] Failed to resolve the IP of `yb-master-0.yb-masters.yb-dev-test.svc.cluster.local' in 6132516ns
2019-01-29 23:46:13.372 WARN  AsyncYBClient.java:1213 [application-akka.actor.default-dispatcher-151] Couldn't resolve this master's address yb-master-0.yb-masters.yb-dev-test.svc.cluster.local:7100
2019-01-29 23:46:13.372 WARN  GetMasterRegistrationReceived.java:229 [application-akka.actor.default-dispatcher-151] Error receiving a response from: yb-master-0.yb-masters.yb-dev-test.svc.cluster.local:7100
2019-01-29 23:46:13.373 ERROR TablesController.java:234 [application-akka.actor.default-dispatcher-151] Failed to get list of tables in universe 5106ca91-214f-4f32-95d7-dcefd270ea9b
org.yb.client.NonRecoverableException: Couldn't find a valid master in (yb-master-0.yb-masters.yb-dev-test.svc.cluster.local:7100), exceptions: [org.yb.client.NonRecoverableException: Couldn't resolve this master's address yb-master-0.yb-masters.yb-dev-test.svc.cluster.local:7100]
	at org.yb.client.GetMasterRegistrationReceived.incrementCountAndCheckExhausted(GetMasterRegistrationReceived.java:149)
	at org.yb.client.GetMasterRegistrationReceived.access$300(GetMasterRegistrationReceived.java:57)
	at org.yb.client.GetMasterRegistrationReceived$GetMasterRegistrationErrCB.call(GetMasterRegistrationReceived.java:231)
	at org.yb.client.GetMasterRegistrationReceived$GetMasterRegistrationErrCB.call(GetMasterRegistrationReceived.java:220)
	at com.stumbleupon.async.Deferred.doCall(Deferred.java:1280)
	at com.stumbleupon.async.Deferred.addCallbacks(Deferred.java:685)
	at org.yb.client.AsyncYBClient.getMasterTableLocationsPB(AsyncYBClient.java:1218)
	at org.yb.client.AsyncYBClient.locateTablet(AsyncYBClient.java:1183)
	at org.yb.client.AsyncYBClient.sendRpcToTablet(AsyncYBClient.java:883)
	at org.yb.client.AsyncYBClient.getTablesList(AsyncYBClient.java:623)
	at org.yb.client.YBClient.getTablesList(YBClient.java:885)
	at org.yb.client.YBClient.getTablesList(YBClient.java:875)
	at com.yugabyte.yw.controllers.TablesController.universeList(TablesController.java:215)
	at v1.Routes$$anonfun$routes$1$$anonfun$applyOrElse$60$$anonfun$apply$60.apply(Routes.scala:1905)
	at v1.Routes$$anonfun$routes$1$$anonfun$applyOrElse$60$$anonfun$apply$60.apply(Routes.scala:1905)
	at play.core.routing.HandlerInvokerFactory$$anon$4.resultCall(HandlerInvoker.scala:157)
	at play.core.routing.HandlerInvokerFactory$$anon$4.resultCall(HandlerInvoker.scala:156)
	at play.core.routing.HandlerInvokerFactory$JavaActionInvokerFactory$$anon$14$$anon$3$$anon$1.invocation(HandlerInvoker.scala:136)
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:73)
	at play.http.HttpRequestHandler$1.call(HttpRequestHandler.java:54)
	at com.yugabyte.yw.controllers.TokenAuthenticator.call(TokenAuthenticator.java:27)
	at play.core.j.JavaAction$$anonfun$7.apply(JavaAction.scala:108)
	at play.core.j.JavaAction$$anonfun$7.apply(JavaAction.scala:108)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:56)
	at play.api.libs.iteratee.Execution$trampoline$.execute(Execution.scala:70)
	at play.core.j.HttpExecutionContext.execute(HttpExecutionContext.scala:48)
	at scala.concurrent.impl.Future$.apply(Future.scala:31)
	at scala.concurrent.Future$.apply(Future.scala:492)
	at play.core.j.JavaAction.apply(JavaAction.scala:108)
	at play.api.mvc.Action$$anonfun$apply$2$$anonfun$apply$5$$anonfun$apply$6.apply(Action.scala:112)
	at play.api.mvc.Action$$anonfun$apply$2$$anonfun$apply$5$$anonfun$apply$6.apply(Action.scala:112)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21)
	at play.api.mvc.Action$$anonfun$apply$2$$anonfun$apply$5.apply(Action.scala:111)
	at play.api.mvc.Action$$anonfun$apply$2$$anonfun$apply$5.apply(Action.scala:110)
	at scala.Option.map(Option.scala:146)
	at play.api.mvc.Action$$anonfun$apply$2.apply(Action.scala:110)
	at play.api.mvc.Action$$anonfun$apply$2.apply(Action.scala:103)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
	at akka.dispatch.BatchingExecutor$BlockableBatch$$anonfun$run$1.apply$mcV$sp(BatchingExecutor.scala:91)
	at akka.dispatch.BatchingExecutor$BlockableBatch$$anonfun$run$1.apply(BatchingExecutor.scala:91)
	at akka.dispatch.BatchingExecutor$BlockableBatch$$anonfun$run$1.apply(BatchingExecutor.scala:91)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:72)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:90)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:39)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:405)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
2019-01-29 23:46:13.373 ERROR ApiResponse.java:21 [application-akka.actor.default-dispatcher-151] Hit error 500, message: {"error":"Couldn't find a valid master in (yb-master-0.yb-masters.yb-dev-test.svc.cluster.local:7100), exceptions: [org.yb.client.NonRecoverableException: Couldn't resolve this master's address yb-master-0.yb-masters.yb-dev-test.svc.cluster.local:7100]"}
2019-01-29 23:46:13.374 INFO  LocalYBClientService.java:27 [application-akka.actor.default-dispatcher-151] Closing client masters=yb-master-0.yb-masters.yb-dev-test.svc.cluster.local:7100.
2019-01-30 00:37:48.449 INFO  DevopsBase.java:99 [application-akka.actor.default-dispatcher-166] Command to run: [bin/ybcloud.sh aws query current-host --metadata_types instance-id vpc-id privateIp region]
2019-01-30 00:37:52.339 INFO  ShellProcessHandler.java:85 [application-akka.actor.default-dispatcher-166] {"error": "Unable to fetch host metadata"}
2019-01-30 00:37:52.340 INFO  DevopsBase.java:99 [application-akka.actor.default-dispatcher-166] Command to run: [bin/ybcloud.sh gcp query current-host]
2019-01-30 00:37:53.009 INFO  ShellProcessHandler.java:85 [application-akka.actor.default-dispatcher-166] {"error": "Host not in GCP."}

**Comment by kmuthukk** [comment]
@bmatican , @ramkumarvs 

seems similar to error in https://github.com/YugaByte/yugabyte-db/issues/696
```
Couldn't resolve this master's address yb-master-0.yb-masters.yb-dev-test.svc.cluster.local:7100]
```

**Comment by sraghavan-eqix** [comment]
As recommended updated the FQDN to match svc.cluster.local.
We tried to set up Yugaware and it was successful. However, we ran into this issue while creating Universe

2019-01-31 17:35:48,375 [INFO] from com.yugabyte.yw.common.ShellProcessHandler in TaskPool-CreateKubernetesUniverse(df87a9a4-a6a6-4aa2-8cbe-60d33ed6db05)-2 -                 "qosClass": "Burstable"
2019-01-31 17:35:48,375 [INFO] from com.yugabyte.yw.common.ShellProcessHandler in TaskPool-CreateKubernetesUniverse(df87a9a4-a6a6-4aa2-8cbe-60d33ed6db05)-2 -             }
2019-01-31 17:35:48,375 [INFO] from com.yugabyte.yw.common.ShellProcessHandler in TaskPool-CreateKubernetesUniverse(df87a9a4-a6a6-4aa2-8cbe-60d33ed6db05)-2 -         }
2019-01-31 17:35:48,375 [INFO] from com.yugabyte.yw.common.ShellProcessHandler in TaskPool-CreateKubernetesUniverse(df87a9a4-a6a6-4aa2-8cbe-60d33ed6db05)-2 -     ],
2019-01-31 17:35:48,376 [INFO] from com.yugabyte.yw.common.ShellProcessHandler in TaskPool-CreateKubernetesUniverse(df87a9a4-a6a6-4aa2-8cbe-60d33ed6db05)-2 -     "kind": "List",
2019-01-31 17:35:48,376 [INFO] from com.yugabyte.yw.common.ShellProcessHandler in TaskPool-CreateKubernetesUniverse(df87a9a4-a6a6-4aa2-8cbe-60d33ed6db05)-2 -     "metadata": {
2019-01-31 17:35:48,376 [INFO] from com.yugabyte.yw.common.ShellProcessHandler in TaskPool-CreateKubernetesUniverse(df87a9a4-a6a6-4aa2-8cbe-60d33ed6db05)-2 -         "resourceVersion": "",
2019-01-31 17:35:48,376 [INFO] from com.yugabyte.yw.common.ShellProcessHandler in TaskPool-CreateKubernetesUniverse(df87a9a4-a6a6-4aa2-8cbe-60d33ed6db05)-2 -         "selfLink": ""
2019-01-31 17:35:48,376 [INFO] from com.yugabyte.yw.common.ShellProcessHandler in TaskPool-CreateKubernetesUniverse(df87a9a4-a6a6-4aa2-8cbe-60d33ed6db05)-2 -     }
2019-01-31 17:35:48,376 [INFO] from com.yugabyte.yw.common.ShellProcessHandler in TaskPool-CreateKubernetesUniverse(df87a9a4-a6a6-4aa2-8cbe-60d33ed6db05)-2 - }
2019-01-31 17:35:48,382 [ERROR] from com.yugabyte.yw.commissioner.SubTaskGroup in TaskPool-3 - Failed to execute task java.util.concurrent.FutureTask@54f3690f, hit error java.lang.RuntimeException: Claim for volume yb-demo-test/yb-master-0 is not bound yet%!!(MISSING)(EXTRA string=datadir0-yb-master-0).
java.util.concurrent.ExecutionException: java.lang.RuntimeException: Claim for volume yb-demo-test/yb-master-0 is not bound yet%!!(MISSING)(EXTRA string=datadir0-yb-master-0)
	at java.util.concurrent.FutureTask.report(FutureTask.java:122)
	at java.util.concurrent.FutureTask.get(FutureTask.java:192)
	at com.yugabyte.yw.commissioner.SubTaskGroup.waitFor(SubTaskGroup.java:160)
	at com.yugabyte.yw.commissioner.SubTaskGroupQueue.run(SubTaskGroupQueue.java:42)
	at com.yugabyte.yw.commissioner.tasks.CreateKubernetesUniverse.run(CreateKubernetesUniverse.java:81)
	at com.yugabyte.yw.commissioner.TaskRunner.run(TaskRunner.java:141)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.RuntimeException: Claim for volume yb-demo-test/yb-master-0 is not bound yet%!!(MISSING)(EXTRA string=datadir0-yb-master-0)
	at com.yugabyte.yw.commissioner.AbstractTaskBase.logShellResponse(AbstractTaskBase.java:103)
	at com.yugabyte.yw.commissioner.tasks.subtasks.KubernetesCommandExecutor.run(KubernetesCommandExecutor.java:175)
	... 5 common frames omitted
2019-01-31 17:35:48,384 [ERROR] from com.yugabyte.yw.commissioner.SubTaskGroupQueue in TaskPool-3 - SubTaskGroup 'HelmInstall : completed 0 out of 1 tasks.' waitFor() returned failed status.
2019-01-31 17:35:48,385 [ERROR] from com.yugabyte.yw.commissioner.tasks.CreateKubernetesUniverse in TaskPool-3 - Error executing task CreateKubernetesUniverse(df87a9a4-a6a6-4aa2-8cbe-60d33ed6db05), error='HelmInstall : completed 0 out of 1 tasks. failed.'
java.lang.RuntimeException: HelmInstall : completed 0 out of 1 tasks. failed.
	at com.yugabyte.yw.commissioner.SubTaskGroupQueue.run(SubTaskGroupQueue.java:51)
	at com.yugabyte.yw.commissioner.tasks.CreateKubernetesUniverse.run(CreateKubernetesUniverse.java:81)
	at com.yugabyte.yw.commissioner.TaskRunner.run(TaskRunner.java:141)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
2019-01-31 17:35:48,389 [DEBUG] from com.yugabyte.yw.models.Universe in TaskPool-3 - Swapped universe df87a9a4-a6a6-4aa2-8cbe-60d33ed6db05:test details to [{"errorString":null,"deviceInfo":null,"universeUUID":"df87a9a4-a6a6-4aa2-8cbe-60d33ed6db05","expectedUniverseVersion":-1,"clusters":[{"uuid":"f1b1390b-ce73-435a-8811-9e35f5bffc19","clusterType":"PRIMARY","userIntent":{"universeName":"test","provider":"e6b28a51-9ad7-400b-9e78-f386d100e7b4","providerType":"kubernetes","replicationFactor":3,"regionList":["659a82f0-333c-4c32-966f-913ca4eab8c9"],"preferredRegion":null,"instanceType":"small","numNodes":3,"ybSoftwareVersion":"1.1.9.0-b18","accessKeyCode":"yugabyte-default","deviceInfo":{"volumeSize":100,"numVolumes":1,"diskIops":null,"storageClass":"standard","mountPoints":null,"ebsType":null},"assignPublicIP":true,"useTimeSync":false,"masterGFlags":{},"tserverGFlags":{},"instanceTags":{}},"placementInfo":{"cloudList":[{"uuid":"e6b28a51-9ad7-400b-9e78-f386d100e7b4","code":"kubernetes","regionList":[{"uuid":"659a82f0-333c-4c32-966f-913ca4eab8c9","code":"us-west","name":"US West","azList":[{"uuid":"1b11289e-4d44-4ae6-ad44-769bf0e4c3aa","name":"us-west-2","replicationFactor":3,"subnet":null,"numNodesInAZ":3,"isAffinitized":true}]}]}]},"index":0}],"nodePrefix":"yb-demo-test","userAZSelected":false,"nodeDetailsSet":[{"nodeIdx":2,"nodeName":"yb-demo-test-n2","nodeUuid":null,"cloudInfo":{"private_ip":null,"public_ip":null,"public_dns":null,"private_dns":null,"instance_type":"small","subnet_id":null,"az":"us-west-2","region":"us-west","cloud":"kubernetes","assignPublicIP":true,"useTimeSync":false},"azUuid":"1b11289e-4d44-4ae6-ad44-769bf0e4c3aa","placementUuid":"f1b1390b-ce73-435a-8811-9e35f5bffc19","state":"ToBeAdded","isMaster":true,"masterHttpPort":7000,"masterRpcPort":7100,"isTserver":true,"tserverHttpPort":9000,"tserverRpcPort":9100,"isRedisServer":true,"redisServerHttpPort":11000,"redisServerRpcPort":6379,"isYqlServer":true,"yqlServerHttpPort":12000,"yqlServerRpcPort":9042,"allowedActions":["DELETE"]},{"nodeIdx":1,"nodeName":"yb-demo-test-n1","nodeUuid":null,"cloudInfo":{"private_ip":null,"public_ip":null,"public_dns":null,"private_dns":null,"instance_type":"small","subnet_id":null,"az":"us-west-2","region":"us-west","cloud":"kubernetes","assignPublicIP":true,"useTimeSync":false},"azUuid":"1b11289e-4d44-4ae6-ad44-769bf0e4c3aa","placementUuid":"f1b1390b-ce73-435a-8811-9e35f5bffc19","state":"ToBeAdded","isMaster":true,"masterHttpPort":7000,"masterRpcPort":7100,"isTserver":true,"tserverHttpPort":9000,"tserverRpcPort":9100,"isRedisServer":true,"redisServerHttpPort":11000,"redisServerRpcPort":6379,"isYqlServer":true,"yqlServerHttpPort":12000,"yqlServerRpcPort":9042,"allowedActions":["DELETE"]},{"nodeIdx":3,"nodeName":"yb-demo-test-n3","nodeUuid":null,"cloudInfo":{"private_ip":null,"public_ip":null,"public_dns":null,"private_dns":null,"instance_type":"small","subnet_id":null,"az":"us-west-2","region":"us-west","cloud":"kubernetes","assignPublicIP":true,"useTimeSync":false},"azUuid":"1b11289e-4d44-4ae6-ad44-769bf0e4c3aa","placementUuid":"f1b1390b-ce73-435a-8811-9e35f5bffc19","state":"ToBeAdded","isMaster":true,"masterHttpPort":7000,"masterRpcPort":7100,"isTserver":true,"tserverHttpPort":9000,"tserverRpcPort":9100,"isRedisServer":true,"redisServerHttpPort":11000,"redisServerRpcPort":6379,"isYqlServer":true,"yqlServerHttpPort":12000,"yqlServerRpcPort":9042,"allowedActions":["DELETE"]}],"updateInProgress":false,"updateSucceeded":false,"nextClusterIndex":1,"importedState":"NONE","capability":"EDITS_ALLOWED"}] with new version = 6.
2019-01-31 17:35:48,391 [DEBUG] from com.yugabyte.yw.commissioner.tasks.UniverseTaskBase in TaskPool-3 - Unlocked universe df87a9a4-a6a6-4aa2-8cbe-60d33ed6db05 for updates.
2019-01-31 17:35:48,391 [ERROR] from com.yugabyte.yw.commissioner.TaskRunner in TaskPool-3 - Error running task
java.lang.RuntimeException: HelmInstall : completed 0 out of 1 tasks. failed.
	at com.yugabyte.yw.commissioner.SubTaskGroupQueue.run(SubTaskGroupQueue.java:51)
	at com.yugabyte.yw.commissioner.tasks.CreateKubernetesUniverse.run(CreateKubernetesUniverse.java:81)
	at com.yugabyte.yw.commissioner.TaskRunner.run(TaskRunner.java:141)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
2019-01-31 17:35:48,391 [INFO] from com.yugabyte.yw.commissioner.TaskRunner in TaskPool-3 - Updating task [taskType : CreateKubernetesUniverse, taskState: Running] to new state Failure
2019-01-31 17:35:48,659 [INFO] from com.yugabyte.yw.commissioner.Commissioner in TaskProgressMonitor - Task task-info {taskType : CreateKubernetesUniverse, taskState: Failure}, task {CreateKubernetesUniverse(df87a9a4-a6a6-4aa2-8cbe-60d33ed6db05) : details={"errorString":null,"deviceInfo":null,"universeUUID":"df87a9a4-a6a6-4aa2-8cbe-60d33ed6db05","expectedUniverseVersion":-1,"clusters":[{"uuid":"f1b1390b-ce73-435a-8811-9e35f5bffc19","clusterType":"PRIMARY","userIntent":{"universeName":"test","provider":"e6b28a51-9ad7-400b-9e78-f386d100e7b4","providerType":"kubernetes","replicationFactor":3,"regionList":["659a82f0-333c-4c32-966f-913ca4eab8c9"],"preferredRegion":null,"instanceType":"small","numNodes":3,"ybSoftwareVersion":"1.1.9.0-b18","accessKeyCode":"yugabyte-default","deviceInfo":{"volumeSize":100,"numVolumes":1,"diskIops":null,"storageClass":"standard","mountPoints":null,"ebsType":null},"assignPublicIP":true,"useTimeSync":false,"masterGFlags":{},"tserverGFlags":{},"instanceTags":{}},"placementInfo":{"cloudList":[{"uuid":"e6b28a51-9ad7-400b-9e78-f386d100e7b4","code":"kubernetes","regionList":[{"uuid":"659a82f0-333c-4c32-966f-913ca4eab8c9","code":"us-west","name":"US West","azList":[{"uuid":"1b11289e-4d44-4ae6-ad44-769bf0e4c3aa","name":"us-west-2","replicationFactor":3,"subnet":null,"numNodesInAZ":3,"isAffinitized":true}]}]}]},"index":0}],"nodePrefix":"yb-demo-test","userAZSelected":false,"nodeDetailsSet":[{"nodeIdx":1,"nodeName":"yb-demo-test-n1","nodeUuid":null,"cloudInfo":{"private_ip":null,"public_ip":null,"public_dns":null,"private_dns":null,"instance_type":"small","subnet_id":null,"az":"us-west-2","region":"us-west","cloud":"kubernetes","assignPublicIP":true,"useTimeSync":false},"azUuid":"1b11289e-4d44-4ae6-ad44-769bf0e4c3aa","placementUuid":"f1b1390b-ce73-435a-8811-9e35f5bffc19","state":"ToBeAdded","isMaster":true,"masterHttpPort":7000,"masterRpcPort":7100,"isTserver":true,"tserverHttpPort":9000,"tserverRpcPort":9100,"isRedisServer":true,"redisServerHttpPort":11000,"redisServerRpcPort":6379,"isYqlServer":true,"yqlServerHttpPort":12000,"yqlServerRpcPort":9042,"allowedActions":["DELETE"]},{"nodeIdx":2,"nodeName":"yb-demo-test-n2","nodeUuid":null,"cloudInfo":{"private_ip":null,"public_ip":null,"public_dns":null,"private_dns":null,"instance_type":"small","subnet_id":null,"az":"us-west-2","region":"us-west","cloud":"kubernetes","assignPublicIP":true,"useTimeSync":false},"azUuid":"1b11289e-4d44-4ae6-ad44-769bf0e4c3aa","placementUuid":"f1b1390b-ce73-435a-8811-9e35f5bffc19","state":"ToBeAdded","isMaster":true,"masterHttpPort":7000,"masterRpcPort":7100,"isTserver":true,"tserverHttpPort":9000,"tserverRpcPort":9100,"isRedisServer":true,"redisServerHttpPort":11000,"redisServerRpcPort":6379,"isYqlServer":true,"yqlServerHttpPort":12000,"yqlServerRpcPort":9042,"allowedActions":["DELETE"]},{"nodeIdx":3,"nodeName":"yb-demo-test-n3","nodeUuid":null,"cloudInfo":{"private_ip":null,"public_ip":null,"public_dns":null,"private_dns":null,"instance_type":"small","subnet_id":null,"az":"us-west-2","region":"us-west","cloud":"kubernetes","assignPublicIP":true,"useTimeSync":false},"azUuid":"1b11289e-4d44-4ae6-ad44-769bf0e4c3aa","placementUuid":"f1b1390b-ce73-435a-8811-9e35f5bffc19","state":"ToBeAdded","isMaster":true,"masterHttpPort":7000,"masterRpcPort":7100,"isTserver":true,"tserverHttpPort":9000,"tserverRpcPort":9100,"isRedisServer":true,"redisServerHttpPort":11000,"redisServerRpcPort":6379,"isYqlServer":true,"yqlServerHttpPort":12000,"yqlServerRpcPort":9042,"allowedActions":["DELETE"]}],"updateInProgress":false,"updateSucceeded":true,"nextClusterIndex":1,"importedState":"NONE","capability":"EDITS_ALLOWED"}} has failed.
2019-01-31 20:37:12,100 [INFO] from com.yugabyte.yw.common.DevopsBase in application-akka.actor.default-dispatcher-31 - Command to run: [bin/ybcloud.sh aws query current-host --metadata_types instance-id vpc-id privateIp region]
2019-01-31 20:37:15,872 [INFO] from com.yugabyte.yw.common.ShellProcessHandler in application-akka.actor.default-dispatcher-31 - {"error": "Unable to fetch host metadata"}
2019-01-31 20:37:15,873 [INFO] from com.yugabyte.yw.common.DevopsBase in application-akka.actor.default-dispatcher-31 - Command to run: [bin/ybcloud.sh gcp query current-host]
2019-01-31 20:37:16,526 [INFO] from com.yugabyte.yw.common.ShellProcessHandler in application-akka.actor.default-dispatcher-31 - {"error": "Host not in GCP."}
